packages/discord.js/src/util/APITypes.js

```diff
@@ -0,0 +1,314 @@
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildExplicitContentFilter}
+ */
+
+/**
+ * @external GuildFeature
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildFeature}
+ */
+
+/**
+ * @external GuildMFALevel
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildMFALevel}
+ */
+
+/**
+ * @external GuildNSFWLevel
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildNSFWLevel}
+ */
+
+/**
+ * @external GuildPremiumTier
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildPremiumTier}
+ */
+
+/**
+ * @external GuildScheduledEventEntityType
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildScheduledEventEntityType}
+ */
+
+/**
+ * @external GuildScheduledEventPrivacyLevel
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildScheduledEventPrivacyLevel}
+ */
+
+/**
+ * @external GuildScheduledEventStatus
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildScheduledEventStatus}
+ */
+
+/**
+ * @external GuildSystemChannelFlags
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildSystemChannelFlags}
+ */
+
+/**
+ * @external GuildVerificationLevel
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/GuildVerificationLevel}
+ */
+
+/**
+ * @external IntegrationExpireBehavior
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/IntegrationExpireBehavior}
+ */
+
+/**
+ * @external InteractionType
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/InteractionType}
+ */
+
+/**
+ * @external InteractionResponseType
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/InteractionResponseType}
+ */
+
+/**
+ * @external InviteTargetType
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/InviteTargetType}
+ */
+
+/**
+ * @external Locale
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-rest/common/enum/Locale}
+ */
+
+/**
+ * @external MessageActivityType
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/MessageActivityType}
+ */
+
+/**
+ * @external MessageType
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/MessageType}
+ */
+
+/**
+ * @external MessageFlags
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/MessageFlags}
+ */
+
+/**
+ * @external OAuth2Scopes
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/OAuth2Scopes}
+ */
+
+/**
+ * @external PermissionFlagsBits
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-payloads/common#PermissionFlagsBits}
+ */
+
+/**
+ * @external RESTJSONErrorCodes
+ * @see {@link https://discord-api-types.dev/api/next/discord-api-types-rest/common/enum/RESTJSONErrorCodes}
```

Comment:
Shouldn't use `next` version in string
```suggestion
 * @see {@link https://discord-api-types.dev/api/discord-api-types-rest/common/enum/RESTJSONErrorCodes}
```